copyparty/web/browser.js

```diff
@@ -1878,9 +1878,636 @@ var Ls = {
+		"wt_pst": "paste a previously cut / copied selection$NHotkey: ctrl-V",
+		"wt_selall": "select all files$NHotkey: ctrl-A (when file focused)",
+		"wt_selinv": "invert selection",
+		"wt_zip1": "download this folder as archive",
+		"wt_selzip": "download selection as archive",
+		"wt_seldl": "download selection as separate files$NHotkey: Y",
+		"wt_npirc": "copy irc-formatted track info",
+		"wt_nptxt": "copy plaintext track info",
+		"wt_m3ua": "add to m3u playlist (click <code>📻copy</code> later)",
+		"wt_m3uc": "copy m3u playlist to clipboard",
+		"wt_grid": "toggle grid / list view$NHotkey: G",
+		"wt_prev": "previous track$NHotkey: J",
+		"wt_play": "play / pause$NHotkey: P",
+		"wt_next": "next track$NHotkey: L",
+
+		"ul_par": "parallel uploads:",
+		"ut_rand": "randomize filenames",
+		"ut_u2ts": "copy the last-modified timestamp$Nfrom your filesystem to the server\">📅",
+		"ut_ow": "overwrite existing files on the server?$N🛡️: never (will generate a new filename instead)$N🕒: overwrite if server-file is older than yours$N♻️: always overwrite if the files are different",
+		"ut_mt": "continue hashing other files while uploading$N$Nmaybe disable if your CPU or HDD is a bottleneck",
+		"ut_ask": 'ask for confirmation before upload starts">💭',
+		"ut_pot": "improve upload speed on slow devices$Nby making the UI less complex",
+		"ut_srch": "don't actually upload, instead check if the files already $N exist on the server (will scan all folders you can read)",
+		"ut_par": "pause uploads by setting it to 0$N$Nincrease if your connection is slow / high latency$N$Nkeep it 1 on LAN or if the server HDD is a bottleneck",
+		"ul_btn": "drop files / folders<br>here (or click me)",
+		"ul_btnu": "U P L O A D",
+		"ul_btns": "S U C H E N",
+
+		"ul_hash": "hash",
+		"ul_send": "senden",
+		"ul_done": "fertig",
+		"ul_idle1": "keine Uploads in der Warteschlange",
+		"ut_etah": "durchschnittl. &lt;em&gt;hashing&lt;/em&gt; Geschw. & gesch. Zeit bis fertig",
+		"ut_etau": "durchschnittl. &lt;em&gt;upload&lt;/em&gt; Geschw. & gesch. Zeit bis fertig",
+		"ut_etat": "durchschnittl. &lt;em&gt;total&lt;/em&gt; Geschw. & gesch. Zeit bis fertig",
+
+		"uct_ok": "Erfolgreich abgeschlossen",
+		"uct_ng": "no-good: fehlgeschlagen / abgelehnt / nicht gefunden",
+		"uct_done": "ok and ng zusammen",
+		"uct_bz": "Hashing oder Uploading",
+		"uct_q": "Bereit, Wartend",
+
+		"utl_name": "Dateiname",
+		"utl_ulist": "Liste",
+		"utl_ucopy": "kopieren",
+		"utl_links": "Links",
+		"utl_stat": "status",
+		"utl_prog": "Fortschritt",
+
+		// keep short:
+		"utl_404": "404",
+		"utl_err": "Fehler",
+		"utl_oserr": "OS-error",
+		"utl_found": "gefunden",
+		"utl_defer": "zurückstellen",
+		"utl_yolo": "YOLO",
+		"utl_done": "fertig",
+
+		"ul_flagblk": "Die Dateien wurden zur Warteschlange hinzugefügt</b><br>jedoch ist up2k gerade in einem anderen Browsertab aktiv.<br>Es wird gewartet, bis diese Aktion abgeschlossen ist.",
```

Comment:
"Es wird gewartet, bis diese Aktion abgeschlossen ist." sounds a bit stilted imo, how about "[...] Ich warte, bis der Upload abgeschlossen ist."? first person is probably fine in the casual tone you're going for in the rest of the translations